### ISSUE TITLE
TypeError: e.filter is not a function

### ISSUE THREAD

**Issue by Kakise** [netlify/netlify-cms#1320]
**- Do you want to request a *feature* or report a *bug*?**
Bug
**- What is the current behavior?**
When entering the Workflow tab, it throws "TypeError: e.filter is not a function"
**- If the current behavior is a bug, please provide the steps to reproduce.**
Enter Workflow tab with at least one draft
**- What is the expected behavior?**
Showing the workflow tab

**- Please mention your versions where applicable.**

Netlify CMS version: 1.7.0
Browser version: Chrome 65

**- Please link or paste your `config.yml` below if applicable.**
Everything is here https://github.com/Kakise/Blog/tree/master/source/admin

**Comment by Kakise** [comment]
Here's the full output of my console:

```Netlify CMS version 1.7.0
5ae08eae3672df314539bf1f--kakise-blog.netlify.com/:1 ServiceWorker Register Successfully.

/favicon.ico Failed to load resource: the server responded with a status of 404 ()

API.js:211  Checking for Unpublished entries
/admin/#/workflow:1 Failed to load https://api.github.com/repos/Kakise/Blog/pulls?ts=1524666201470&head=cms%2Fpoint-sur-le-cloud-gaming-avec-liquidsky&state=open&base=master: Redirect from 'https://api.github.com/repos/Kakise/Blog/pulls?ts=1524666201470&head=cms%2Fpoint-sur-le-cloud-gaming-avec-liquidsky&state=open&base=master' to 'https://api.github.com/repos/Kakise/Blog/pulls?ts=1524666201470&head=anonymized&state=open&base=master' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://5ae08eae3672df314539bf1f--kakise-blog.netlify.com' is therefore not allowed access. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
API.js:229  No Unpublished entries

backend.js:192 Uncaught (in promise) TypeError: e.filter is not a function
    at backend.js:192```

**Comment by Kakise** [comment]
After I cleaned my repository and branches, everything went back to normal. Seems like the issue is repository related

**Comment by erquhart** [comment]
Yeah, sounds about right. We do need to improve some of the workflow handling as this kind of issue isn't uncommon.

**Comment by avigoldman** [comment]
@erquhart any suggestions for fixing this beyond deleting all the branches?

**Comment by erquhart** [comment]
Metadata may have been damaged by earlier 2.x releases for folks that ran them (fixed now). Unfortunately we don't know enough about the bug when it existed to have a fix for it. If you can point to a repo I can take a look.

**Comment by avigoldman** [comment]
Glad to hear that it's fixed for the future!

I ended up pulling down all the open PR branches, deleting them in GitHub, repushing and recreating all the PRs, which seemed to fix the issue.

**Comment by erquhart** [comment]
Cool, glad you got it solved. In the future I expect us to drop the mysterious metadata stuff altogether and just track PR's explicitly.

**Comment by nop33** [comment]
@Kakise 
> After I cleaned my repository and branches, everything went back to normal. Seems like the issue is repository related

Thank you so much XD

**Comment by abstracthat** [comment]
I just ran into this issue and indeed removing all branches/PR's did in fact clear it up.

**Comment by erquhart** [comment]
Closing this up, please open a new issue with details if you encounter this.
